roles/openshift_logging/tasks/start_cluster.yaml

```diff
@@ -37,9 +39,11 @@
   changed_when: no
 
 - name: start kibana
-  include: scale.yaml
-  vars:
-    desired: "{{ openshift_logging_kibana_replica_count | default (1) }}"
+  oc_scale:
+    kind: dc
+    name: "{{object.split('/')[1]}}"
+    namespace: "{{mktemp.stdout}}/admin.kubeconfig"
```

Comment:
same as above.  I also noticed others here.  Probably a search and replace should fix this.